### ISSUE TITLE
Hi, how could i save some statistics like training loss and accuracy to a file?

### ISSUE THREAD

**Issue by lfwin** [tflearn/tflearn#404]
this maybe very helpful to use these data to plot my own graph.

**Comment by aymericdamien** [comment]
Yes, tensorboard can export all data you need. Otherwise, you need to modify tflearn source (that file https://github.com/tflearn/tflearn/blob/master/tflearn/callbacks.py#L81) and save data to a file.
